src/test/java/rest/salesforce/org/steps/OrdersApiSteps.java

```diff
@@ -34,14 +73,25 @@ public void deleteOrder() {
 
     @Given("I build a {string} request for a single Order")
     public void iBuildARequestForASingleOrder(final String apiMethod) {
+        requestBuilder
+                .addToken(dotenv.get("TOKEN"))
+                .addBaseUri(dotenv.get("BASE_URL"))
+                .addMethod(ApiMethod.GET);
     }
 
     @When("I execute the get single Order request on {string} endpoint")
     public void iExecuteTheGetSingleOrderRequestOnEndpoint(final String endpoint) {
```

Comment:
Parameter endpoint is never used, it should be deleted.

Reply:
ok